src/@types/MetaData.d.ts

```diff
@@ -39,6 +39,19 @@ export interface MetadataPublishForm {
   links?: string | File[]
 }
 
+export interface AlgorithmPublishForm {
```

Comment:
can't think of any naming for both interfaces here making the 2 types consistent without making it super long, like:

- `MetadataPublishFormDataset`
- `MetadataPublishFormAlgorithm`

Or is not too long? On other hand kinda clear that publish form holds metadata so maybe we can get a way with:

- `PublishFormDataset`
- `PublishFormAlgorithm`

but that might collide with our publish forms component naming

Reply:
I believe using `MetadataPublishFormDataset` and `MetadataPublishFormAlgorithm` will be more consistent and more clear what that form holds, even if it's a bit too long